### PR TITLE
format css, add 2 media queries on the navbar

### PR DESCRIPTION
**Pull request by ChristopherDalziel**

### CODE REVIEW

**Comment by ChristopherDalziel** [comment]
Hi - 

I had to format your css file so I could actually read it, I'm not sure if there is any reason to single line it because of Gatsby? Decline the PR if so! I tried to install Gatsby but after spending 45 minutes updating brew and node I still failed so I decided to just put some code in and let you play with it. 

```
// -> max-width 550px = any screen size at 550px wide or below the css within the {} will apply
@media (max-width: 550px) {
// Normal css selectors here -> can be ANY amount
  .navbar {
// Consider the css within here that exits above to be an ovveride, a media query has more specificy then a normal css selection unless you use !important or maybe even if the selector is more specific. For example this will make the background blue once the screen is sub 550px you could also add new styles here doesn't only have to be overides, for example maybe sub 550px you want no navbar?
    background-color: blue;
  }
}
```

I only added two and they aren't going to be very functional I just wanted to prove you could make the nav smaller but I didn't consider the buttons etc at all so it may break or it may not even work. The color changes should work at least which is why I put those in too. A better example would have been the project cards/boxes but I'd be really really winging it. 

Doc about @media ->
https://www.w3schools.com/cssref/css3_pr_mediaquery.asp 

Something else that is cool ->
CSS in JS
https://emotion.sh/docs/styled 

Basically means we throw standard html tags in the bin, we now write custom styled components and call those instead, these components can and should come with pre-attached styles, that can also be adjusted by passing down different properties as you would a normal React component. It is a very common way for people to style React components as it makes each component even more tightly wrapped if you will since it's carrying it's own style as well as the component itself. I'm not sure this would work with Gatsby etc regardless, might have to look at npx create-react-app next ;)
